rcljava/src/main/java/org/ros2/rcljava/RCLJava.java

```diff
@@ -255,13 +255,37 @@ public static Node createNode(final String nodeName) {
    * @return A @{link Node} that represents the underlying ROS2 node
    *     structure.
    */
+  public static Node createNode(final String nodeName, final String namespace) {
+    return createNode(nodeName, namespace, new NodeOptions());
+  }
+
+  /**
+   * Create a @{link Node}.
+   *
+   * @param nodeName The name that will identify this node in a ROS2 graph.
+   * @param namespace The namespace of the node.
+   * @param context Context used for creating the node, @link{RCLJava.getDefaultContext()} will be used if `null`.
+   * @return A @{link Node} that represents the underlying ROS2 node
+   *     structure.
```

Comment:
nit: this doesn't have to be on a new line (same comment for other doc-block below).

Reply:
done in https://github.com/osrf/ros2_java/pull/46/commits/1d28970b84526a67303e210ab3090431338dba1f.